Dockerfile

```diff
@@ -136,6 +139,23 @@ COPY scripts/async_profiler_build_shared.sh .
 COPY scripts/async_profiler_build_musl.sh .
 RUN ./async_profiler_build_shared.sh /tmp/async_profiler_build_musl.sh
 
+# node-package-builder-musl
+FROM alpine${NODE_PACKAGE_BUILDER_MUSL} AS node-package-builder-musl
+WORKDIR /tmp
+COPY scripts/build_node_package.sh .
+RUN apk add --no-cache curl g++ python3 make gcc git bash nodejs npm
+RUN ./build_node_package.sh
+
+# node-package-builder-glibc
+FROM ubuntu${NODE_PACKAGE_BUILDER_GLIBC} AS node-package-builder-glibc
+WORKDIR /tmp
+COPY scripts/build_node_package.sh .
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update -y && apt install -y curl g++ python3 make gcc git
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
```

Comment:
Btw why do we need this here but not for the alpine? Because the alpine has recent `nodejs` package, while the ubuntu one does not?

Reply:
Ubuntu installs node 10, which is too old for node-gyp. In case of alpine default one is 14.

Reply:
I see, please comment on the Dockerfile :)